src/jit/lclvars.cpp

```diff
@@ -6533,9 +6533,15 @@ void Compiler::lvaDumpRegLocation(unsigned lclNum)
 #ifdef _TARGET_ARM_
     else if (varDsc->TypeGet() == TYP_DOUBLE)
     {
+#ifdef LEGACY_BACKEND
+        // The assigned registers are `lvRegNum:lvOtherReg`
         printf("%3s:%-3s    ", getRegName(varDsc->lvRegNum), getRegName(varDsc->lvOtherReg));
```

Comment:
@BruceForstall First reason for displaying double regs as `f4:f5` instead of `d2` is this is how legacy backend does. And I thought `d2` could be confusing since arm disasm prints `d4` for `f4:f5`. So I thought just printing `f4:f5` would make less confusion.